addons/product/migrations/9.0.1.2/pre-migration.py

```diff
@@ -6,6 +6,9 @@
 
 from openupgradelib import openupgrade
 
+table_renames = [
```

Comment:
I think you don't need to rename the table, as it's not dropped during upgrade process.